carma-messenger-core/hooks/pre_build

```diff
@@ -6,8 +6,8 @@ if [[ "$SOURCE_BRANCH" = "develop" ]]; then
         carma-messenger-core/Dockerfile
 elif [[ "$SOURCE_BRANCH" =~ ^release/.*$ ]]; then
     # swap checkout branch in checkout.bash to release branch
-    sed -i "s|usdotfhwastoldev|usdotfhwastolcandidate|g; s|usdotfhwastol|usdotfhwastolcandidate|g; s|:[0-9]*\.[0-9]*\.[0-9]*|:candidate|g; s|CARMA[a-zA-Z]*_[0-9]*\.[0-9]*\.[0-9]*|candidate|g; s|carma-[a-zA-Z]*-[0-9]*\.[0-9]*\.[0-9]*|candidate|g; s|develop|candidate|g" \
-        carma-messenger-core/Dockerfile
+    sed -i "s|usdotfhwastoldev/|usdotfhwastolcandidate/|g; s|usdotfhwastol/|usdotfhwastolcandidate/|g; s|:[0-9]*\.[0-9]*\.[0-9]*|:candidate|g; s|CARMA[a-zA-Z]*_[0-9]*\.[0-9]*\.[0-9]*|candidate|g; s|carma-[a-zA-Z]*-[0-9]*\.[0-9]*\.[0-9]*|candidate|g; s|develop|candidate|g" \
+        Dockerfile
```

Comment:
I think we'd want to keep the path here to be `carma-messenger-core/Dockerfile`, since there isn't a `Dockerfile` and `checkout.bash` in the root?